airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/legacy_typing_deduping/TypingDedupingFinalTableOperations.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration.legacy_typing_deduping
+
+import io.airbyte.cdk.load.command.DestinationStream
+import io.airbyte.cdk.load.orchestration.ColumnNameMapping
+import io.airbyte.cdk.load.orchestration.DestinationHandler
+import io.airbyte.cdk.load.orchestration.TableName
+import io.airbyte.cdk.load.orchestration.TableNames
+import io.airbyte.cdk.load.orchestration.TableNames.Companion.SOFT_RESET_SUFFIX
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.time.Instant
+
+private val logger = KotlinLogging.logger {}
+
```

Comment:
this is the half of the old CDK's StorageOperation class that deals with final tables. I split it out b/c there's nothing destination-specific in here, all the implementations were identical